Makefile

```diff
@@ -157,9 +165,47 @@ cmctl: ## Downloads and installs cmctl, the CLI for cert-manager, to your local
 	rm -rf $$TMP_DIR ;\
 	}
 
+.PHONY: cert-manager
+cert-manager: cmctl ## Ensures cert-manager is deployed in the current Kubernetes cluster and verifies API access with kubectl wait
```

Comment:
Please move comments before target just like in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/Makefile#L335

Reply:
The 'make help' action that prints out information about how to use this makefile relies on comments being in this location. I can try refactoring 'make help' and the format of comments in this Makefile in a separate PR. 

Here's an example os using 'make help`.
> % make help
> 
> Usage:
>   make <target>
> 
> General
>   help             Display Makefile help information for all actions
> 
> Initialization
>   install-tools    Install tools (macOS/Linux)
> 
> Build
>   dep-update       Fetch Helm chart dependency repositories, build the Helm chart with the dependencies specified in the Chart.yaml
>   render           Render the Helm chart with the examples as input. Users can also provide value overrides.
> ...